packages/react-components/src/molecules/SearchField.tsx

```diff
@@ -32,8 +32,10 @@ type SearchProps = Pick<
   ComponentProps<typeof TextField>,
   'value' | 'onChange'
 > &
-  Required<Pick<ComponentProps<typeof TextField>, 'placeholder'>>;
-const SearchField: React.FC<SearchProps> = (props) => {
+  Required<Pick<ComponentProps<typeof TextField>, 'placeholder'>> & {
+    padding?: boolean;
+  };
+const SearchField: React.FC<SearchProps> = ({ padding = 'true', ...props }) => {
```

Comment:
padding is a boolean, but we've defaulted it to a string. Shouldn't the typing have complained?

Reply:
yeah idk how it doesnt complain about this one